docs/v0.0.1/layout-specs.md

```diff
@@ -41,90 +82,182 @@ Each element in the hierarchical structure contains the following key-value pair
+          "include": [
+            {
+              "path": "DSCore.Color",
+              "iconUrl": "/src/resources/icons/DSCore.Color.png"
+            },
+            {
+              "path": "DSCore.ColorRange",
+              "iconUrl": "/src/resources/icons/DSCore.ColorRange.png"
+            }
+          ],
+          "childElements": [
+            {
+              "text": "Watch",
+              "iconUrl": "Watch",
+              "elementType": "group",
+              "include": []
+            }   
+          ]
+        }]
     }]
 }
 ```
+Since `Watch` does not have any child items, it will not be rendered in the library. For the next step, we will add some leaf nodes into `Watch`.
 
-This results in library view that looks like the following image:
-
-![image](img/layout-nested-empty.png?raw=true)
-
-### Adding items to a nested element
+### Adding leaf items to a nested element
 The following example adds three new data types under `Watch` element:
 
 ```json
 {
-  "elements": [
+  "sections": [
     {
-      "text": "Display",
-      "iconUrl": "Category.Display.svg",
-      "elementType": "category",
-      "include": [
-        { "path": "DSCore.Color" },
-        { "path": "DSCore.ColorRange2D" }
-      ],
+      "text": "default",
+      "iconUrl": "",
+      "elementType": "section",
+      "showHeader": false,
+      "include": [],
       "childElements": [
         {
-          "text": "Watch",
-          "iconUrl": "Watch",
-          "elementType": "none",
+          "text": "Display",
+          "iconUrl": "/src/resources/icons/Category.Display.svg",
+          "elementType": "category",
           "include": [
-            { "path": "Core.View.Watch" },
-            { "path": "Core.View.Watch Image" },
-            { "path": "Core.View.Watch 3D" }            
+            {
+              "path": "DSCore.Color",
+              "iconUrl": "/src/resources/icons/DSCore.Color.png"
+            },
+            {
+              "path": "DSCore.ColorRange",
+              "iconUrl": "/src/resources/icons/DSCore.ColorRange.png"
+            }
+          ],
+          "childElements": [
+            {
+              "text": "Watch",
+              "iconUrl": "Watch",
+              "elementType": "group",
+              "include": [
+                { "path": "Core.View.Watch" },
+                { "path": "Core.View.Watch Image" },
+                { "path": "Core.View.Watch 3D" } 
+              ]
+            }   
           ]
-        }        
-      ]
+        }]
     }]
 }
 ```
 
 This results in library view that looks like the following image:
 
-![image](img/layout-nested-items.png?raw=true)
+![image](img/layout-leaf-items.png?raw=true)
+
+### The Miscellaneous section
+As mentioned, this layout specification works closely with `loadedTypes`. The `Miscellaneous` section, which displays left-over items, only shows up if `loadedTypes` contains items that are unspecified in this layout specification.
```

Comment:
All items in `loadedTypes` that are included through layout specification will show up under the `default` section on the library, as for ones that are not included, they will be displayed under `Miscellaneous` section.